File: skills/astra-toolkit/clients/go/examples/tables-drop-table-exists.go
package main

import (
	"context"
	"log"
	"os"

	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	ctx := context.Background()

	// Get a database
	client := astra.NewClient()

	database := client.Database(
		os.Getenv("ASTRA_DB_API_ENDPOINT"),
		options.API().SetToken(os.Getenv("ASTRA_DB_APPLICATION_TOKEN")),
	)

	// Drop a table
	err := database.DropTable(
		ctx,
		"**TABLE_NAME**",
		options.DropTable().SetIfExists(true),
	)
	if err != nil {
		log.Fatal(err)
	}
}

File: skills/astra-toolkit/clients/csharp/examples/tables-drop-table-exists.cs
using DataStax.AstraDB.DataApi;
using DataStax.AstraDB.DataApi.Core;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    // Get an existing database
    var client = new DataAPIClient();
    var database = client.GetDatabase(
      System.Environment.GetEnvironmentVariable("ASTRA_DB_API_ENDPOINT"),
      System.Environment.GetEnvironmentVariable("ASTRA_DB_APPLICATION_TOKEN")
    );

    // Drop a table
    await database.DropTableAsync(
      "**TABLE_NAME**",
      new DropTableOptions() { IfExists = true }
    );
  }
}
